frontend/src/scenes/insights/yAxisFormat.ts

```diff
@@ -0,0 +1,31 @@
+import { LemonSelectOptions } from 'lib/components/LemonSelect'
+import { compactNumber, humanFriendlyDuration } from 'lib/utils'
+
+const formats = ['numeric', 'duration', 'percentage'] as const
+export type YAxisFormat = typeof formats[number]
+
+export const isYAxisFormat = (candidate: unknown): candidate is YAxisFormat =>
+    formats.includes(candidate as YAxisFormat)
+
+export const yAxisFormatSelectOptions = formats.reduce((target, format) => {
+    target[format as string] = { label: format as string }
+    return target
+}, {} as LemonSelectOptions)
+
+const toPercentage = (value: number | string): string => {
```

Comment:
We do already have a `percentage` util, though that one is intended for values in the 0-1 range. Do you think we'll be able to reuse it by just multiplying with 100?